packages/p2p/src/components/my-profile/my-profile-separator-container/my-profile-separator-container.jsx

```diff
@@ -3,10 +3,11 @@ import * as React from 'react';
 
 const MyProfileSeparatorContainer = ({ children }) => <div className='my-profile-separator-container'>{children}</div>;
 
-const MyProfileSeparatorContainerLine = ({ is_invisible }) => (
+const MyProfileSeparatorContainerLine = ({ className, is_invisible }) => (
     <div
         className={classNames('my-profile-separator-container__line', {
```

Comment:
```suggestion
        className={classNames('my-profile-separator-container__line', className, {
```
I think we should add the `className` here. Not sure if would if we put it as a prop in the object.

Reply:
Sure.